apps/web/src/pages/index.tsx

```diff
@@ -1,14 +1,34 @@
+import { useEffect } from "react";
 import BridgeForm from "@components/BridgeForm";
 import WelcomeHeader from "@components/WelcomeHeader";
 // import MobileBottomMenu from "@components/MobileBottomMenu";
 import useWatchEthTxn from "@hooks/useWatchEthTxn";
 import TransactionStatus from "@components/TransactionStatus";
 import { useTransactionHashContext } from "@contexts/TransactionHashContext";
 import { CONFIRMATIONS_BLOCK_TOTAL } from "../constants";
+import useBridgeFormStorageKeys from "../hooks/useBridgeFormStorageKeys";
+import { getStorageItem } from "../utils/localStorage";
 
 function Home() {
   const { ethTxnStatus, isApiSuccess } = useWatchEthTxn();
   const { txnHash, setTxnHash } = useTransactionHashContext();
+  const { UNCONFIRMED_TXN_HASH_KEY } = useBridgeFormStorageKeys();
+
+  useEffect(() => {
+    const unloadCallback = (e) => {
+      const event = e;
+      const unconfirmedHash = getStorageItem<string>(UNCONFIRMED_TXN_HASH_KEY);
+      if (unconfirmedHash !== undefined) {
```

Comment:
```suggestion
      const unconfirmedHash = getStorageItem<string>(UNCONFIRMED_TXN_HASH_KEY);
      const unsentFundHash = getStorageItem<string>(UNSENT_FUND_TXN_HASH_KEY);
      if (unconfirmedHash !== undefined || unsentFundHash !== undefined) {
```
Let's also add a warning for this state

Reply:
Alright done!